MUICore/setup.py

```diff
@@ -10,21 +10,15 @@
 setuptools.setup(
```

Comment:
I'd like to see some/most of the helper bash scripts integrated into this `setup.py` file for this PR as well. I did some quick searching and looks like we can call scripts from this file: https://stackoverflow.com/questions/66816295/how-to-generate-python-code-during-a-build-and-include-those-in-a-python-wheel

For installation into `MUI/os/linux_x86_64/muicore_server` I think a `justfile` with two targets organized like the following would be very nice to have for repeating the processes you are doing:

```justfile
build:
  ...

install: build
  ...
```

I'm hoping that each of `build` and `install` is only 1 command like `python3 setup.py build` (for (re)generating the grpc/protobuf files)  and `MUICore/build_binary.sh`, respectively.

The naming of the script/targets might be a little confusing right now... maybe change the `justfile` `build` target to `generate`? Maybe have 3 `justfile` targets: `generate`, `build`, `install`, where `generate` regenerates protobuf/grpc, `build` calls shiv to create the binary, and `install` copies the shiv-built binary to MUI Ghidra directory.

```justfile
generate:
  ...

build: generate
  ...

install: build
  ...
```